### PR TITLE
fix: Smooth SLV scrolling

### LINKED ISSUE #2035: [NGRM] - Scrolling is erratic when flattening in sync log viewer
Scrolling in sync log view behaves erratically (where it should be smoothed) if the wells are flattened according to well pick. This can also be observed in internal Webviz storybook: 
https://equinor.github.io/webviz-subsurface-components/storybook-static/?path=/story/welllogviewer-demo-synclogviewer--default&args=syncTrackPos:!false;syncContentSelection:!false

### PR DESCRIPTION
**Pull request by Vladimir-Kokin**
Make scrolling with a scroller to be smooth

Closes #2035

### CODE REVIEW

File: typescript/packages/well-log-viewer/src/SyncLogViewer.tsx [hunk 1]
```diff
@@ -540,6 +540,7 @@ class SyncLogViewer extends Component<SyncLogViewerProps, State> {
 
         if (this._isMounted) this.setState({ primaryAxis: value });
     }
+    /*
```
Comment (mirisb) [inline typescript/packages/well-log-viewer/src/SyncLogViewer.tsx:543]: If the code is commented out, can it be removed as to not pollute the code?

**Review by mirisb — APPROVED** [review]

File: typescript/packages/well-log-viewer/src/SyncLogViewer.tsx (hunk 1 above)
Comment (Vladimir-Kokin) [reply typescript/packages/well-log-viewer/src/SyncLogViewer.tsx]: I have removed it

**Comment by Vladimir-Kokin** [comment]
Storybook Smoke Tests are unstable
I had need to rerun to make they passed

**Pull request merged**